src/main/java/uk/gov/register/db/IndexQueryDAO.java

```diff
@@ -237,5 +240,14 @@
             "  index_entry.\"key\"  " +
             "order by  " +
             "  ien  ";
+    
+    String recordCountQuery = "select " +
```

Comment:
We could perhaps make this inline rather than create another block for it down here, given that this isn't a particularly complex query like some of the others in this DAO.